### PR TITLE
Turtlelord26 #491 explosive damage consistency

### PR DESCRIPTION
**Pull request by Turtlelord26**
Offending melee weapons now deal normal damage, which will be clarified in a subsequent pull. Explosive damage is now Explosive damage and not also Concussive damage, which is dead.
#491 
EDIT: now also involves #577 via sub-pull.

### CODE REVIEW

File: Items/04_01_Melee_&_Thrown_Weapons.txt [hunk 1]
```diff
@@ -169,7 +166,7 @@ THROWN WEAPONS
 
 == Gurber Throwing Knife $80 ==
 
-Damage: 15 + STR + DEX + [Thrown Weapon Skill]/2 + 1xBleed
+Damage: 15 + STR + DEX + 1xBleed
```
Comment (bleehu) [inline Items/04_01_Melee_&_Thrown_Weapons.txt]: This constitutes a nasty damage debuff. Can we up the base damage some?

File: Items/04_01_Melee_&_Thrown_Weapons.txt [hunk 2]
```diff
@@ -185,7 +182,7 @@ AP Level: 2
 
 == EAL Tactical Throwing Axe $160 ==
 
-Damage: 25 + 3*STR + DEX + [Thrown Weapon Skill]/2 + 1xBleed
+Damage: 25 + 3*STR + DEX + 1xBleed
```
Comment (bleehu) [inline Items/04_01_Melee_&_Thrown_Weapons.txt]: Same here; this was also written before we limited skills to a 6 point/level cap. We need to amp the damage on each of these by a bit.

**Review by bleehu — CHANGES_REQUESTED** [review]
Simply removing the skill damage bonus from thrown weapons without putting any damage back is pulling them too far below the power curve. Simple fix: also bump their damage by around 15-20 points depending on level of weapon.

**Comment by Turtlelord26** [comment]
Well, I've been meaning to go over all of the melee weapon damages. Will make a PR to this PR.

File: 01_04_Damage_Types_and_Armor_Types.txt [hunk 3]
```diff
@@ -22,10 +22,10 @@ ballistic armor is hit with a laser, they receive damage as if they weren't
 wearing any armor.
 
 Explosives:
-    Certain grenades and devices have Explosive (concussive) damage. Explosive 
-Damage ignores armor and shields (explosive damage is essentially, a crushing 
-force). Fragmentation grenades are considered ballistic weapons, as their 
-primary damage delivery method is shrapnel. 
+    Certain grenades and devices have Explosive damage, which ignores armor and
```
Comment (rosebrallier) [inline 01_04_Damage_Types_and_Armor_Types.txt]: Scope creep: Do we want to standardize to weapons "having" or "dealing" damage types? I personally prefer "dealing".

File: Items/04_01_Melee_&_Thrown_Weapons.txt [hunk 4]
```diff
@@ -117,11 +117,9 @@ Range: 2m
 Accuracy Modifier: +0
 STR Requirement: 6
 
-	Considered concussive (explosive) damage. Hits armor points 
-before physical damage if the target's armor has defense against explosive (i.e. 
-concussive) damage. If armor roll is greater than 50, target rolls a DC 50 
+	If armor roll is greater than 50, target rolls a DC 50 
 fort save for a broken limb of your choice. -20 stealth rolls for being quiet
```
Comment (rosebrallier) [inline Items/04_01_Melee_&_Thrown_Weapons.txt]: Scope creep: needs more clarity re results of save on success vs failure
May want to specify that the "you" here is the wielder.
What even are our rules for broken limbs?

File: Items/04_01_Melee_&_Thrown_Weapons.txt [hunk 5]
```diff
@@ -117,11 +117,9 @@ Range: 2m
 Accuracy Modifier: +0
 STR Requirement: 6
 
-	Considered concussive (explosive) damage. Hits armor points 
-before physical damage if the target's armor has defense against explosive (i.e. 
-concussive) damage. If armor roll is greater than 50, target rolls a DC 50 
+	If armor roll is greater than 50, target rolls a DC 50 
 fort save for a broken limb of your choice. -20 stealth rolls for being quiet 
-on stealth attacks. Reduces door-breaking DC by 30 per hit.
+on stealth attacks. Reduces break DCs of doors by 30 per hit.
```
Comment (rosebrallier) [inline Items/04_01_Melee_&_Thrown_Weapons.txt]: "break DCs" is super awk wording. Re my conventions this should become "the DC to break down a target door"

File: Items/04_01_Melee_&_Thrown_Weapons.txt [hunk 6]
```diff
@@ -131,11 +129,10 @@ Range: Adjacent (1m)
 Accuracy Modifier: +1
 STR Requirement: 5
 
-	Considered concussive (explosive) damage. Hits armor points before physical 
-damage if the target has defense against explosive (i.e. concussive) damage. If 
-armor roll is less than 50%, target rolls a DC 50% fort save for a being stunned 
-for 1 turn. -30% stealth rolls for being quiet on stealth attacks. Reduces 
-door-breaking DC by 45% per hit. This weapon has a Guard modifier of 0.
+	If armor roll is less than 50, target must pass a DC 50 fort save 
+or be stunned for 1 turn. -30 stealth rolls for being quiet on stealth
```
Comment (rosebrallier) [inline Items/04_01_Melee_&_Thrown_Weapons.txt:150]: Note to self: "-30 to Stealth checks"? Is Stealth a skill that I've specified? B/c if not I should do that. Also occurs above.

**Review by rosebrallier — APPROVED** [review]
One sentence structure note, plus comments I'll happily make into new issues. Later.
Shipit pending blee's balance comments.

File: Items/04_01_Melee_&_Thrown_Weapons.txt (hunk 6 above)
Comment (Turtlelord26) [reply Items/04_01_Melee_&_Thrown_Weapons.txt:150]: https://github.com/bleehu/Compound_X/blob/dev/CharacterCreation/06_Skills.txt#L75

File: 01_04_Damage_Types_and_Armor_Types.txt (hunk 3 above)
Comment (Turtlelord26) [reply 01_04_Damage_Types_and_Armor_Types.txt]: Agree

File: Items/04_01_Melee_&_Thrown_Weapons.txt (hunk 4 above)
Comment (Turtlelord26) [reply Items/04_01_Melee_&_Thrown_Weapons.txt]: We don't have them. This weapon is ancient.

File: Items/04_01_Melee_&_Thrown_Weapons.txt (hunk 6 above)
Comment (rosebrallier) [reply Items/04_01_Melee_&_Thrown_Weapons.txt:150]: Oh good! I... maybe need to use the sleeps.

File: Items/04_01_Melee_&_Thrown_Weapons.txt [hunk 7]
```diff
@@ -131,11 +129,10 @@ Range: Adjacent (1m)
...
 STR Requirement: 5
 
-	Considered concussive (explosive) damage. Hits armor points before physical 
-damage if the target has defense against explosive (i.e. concussive) damage. If 
-armor roll is less than 50%, target rolls a DC 50% fort save for a being stunned 
-for 1 turn. -30% stealth rolls for being quiet on stealth attacks. Reduces 
-door-breaking DC by 45% per hit. This weapon has a Guard modifier of 0.
+	If armor roll is less than 50, target must pass a DC 50 fort save 
+or be stunned for 1 turn. -30 stealth rolls for being quiet on stealth 
+attacks. Reduces door-breaking DC by 45 per hit. This weapon has a
```
Comment (rosebrallier) [inline Items/04_01_Melee_&_Thrown_Weapons.txt:151]: Nother door reference. Also Stealth.

File: Items/04_01_Melee_&_Thrown_Weapons.txt [hunk 8]
```diff
@@ -117,11 +117,9 @@ Range: 2m
 Accuracy Modifier: +0
 STR Requirement: 6
 
-	Considered concussive (explosive) damage. Hits armor points 
-before physical damage if the target's armor has defense against explosive (i.e. 
-concussive) damage. If armor roll is greater than 50, target rolls a DC 50 
-fort save for a broken limb of your choice. -20 stealth rolls for being quiet 
-on stealth attacks. Reduces door-breaking DC by 30 per hit.
+	If armor roll is greater than 50, target rolls a DC 50 Fort save 
+for a broken limb of your choice. -20 to stealth rolls for being quiet
```
Comment (rosebrallier) [inline Items/04_01_Melee_&_Thrown_Weapons.txt:138]: Is the Fort save to avoid the borked limb, keep composure at the borken ouch, smth else? We should be explicit abt the results of successful and failed saves.

**Review by rosebrallier — COMMENTED** [review]
Saw more thing.

**Review by bleehu — APPROVED** [review]
shipit!
